docs/visualization/visualize_unit_metrics.ipynb

```diff
@@ -107,14 +107,14 @@
      "name": "stderr",
```

Comment:
<p>I am guessing there is no change in the code below?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/openscope_databook/pull/139/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>